Hosts_Prefix.pch

```diff
@@ -5,3 +5,8 @@
 #ifdef __OBJC__
     #import <Cocoa/Cocoa.h>
 #endif
+
+// In case we run the Hosts code as a Pref Pane, NSLocalizedString does not help
+#define NSLocalizedStringUniversal(key, comment) \
+[[NSBundle bundleForClass: [self class]] localizedStringForKey:(key) value:@"" table:nil]
```

Comment:
Huh I don't understand why both declaration are the same??? Why not use [[NSBundle bundleForClass: directly?

Reply:
In order to use the same code for app and pane, I put NSLocalizedStringUniversal into both .pch files. I did not want to make the same declaration in several files or introduce another header. But I can use NSBundle bundleForClass directly, sure.

Reply:
Removed NSLocalizedStringUniversal